packages/client/src/upload.ts

```diff
@@ -87,6 +96,27 @@ export async function* upload(
 
         const { data } = file;
 
+        uploadProgress.bytesUploaded = 0;
+
+        // Split out into chunks
+        const body = new Readable();
+        const read = body.read;
```

Comment:
**suggestion (non-blocking):** rename `read` to `originalRead`

Can we call this `originalRead` (or something like it) to make it clearer what it is?

Reply:
Updated